### PR TITLE
Managed services definition added  

### PR DESCRIPTION
**Pull request by ShivamTyagi12345**
Here is a definition of Managed services with a detailed focus on the major IT problems it focuses on and how it helps in solving those problems.

I would be happy to incorporate any further amends, once reviewed .

### CODE REVIEW

**Comment by CathPag** [comment]
Thanks, @ShivamTyagi12345. Please give us a little time to review and get back with feedback.

**Comment by ShivamTyagi12345** [comment]
Sure thing @CathPag :)

File: definitions/managed sevices.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Managed sevices
+status: Feedback Appreciated
+category: property
+---
+## Managed sevices
+### What it is
+ A complete safety net to your technical environment. It is the trust that you can have in a team to look after, both proactively and reactively, an ever-changing database environment. Above all, it is the knowledge that you have a team of experts to cover all areas of issues!
```
Comment (JasonMorgan) [inline definitions/managed sevices.md:8]: A little too marketing-y. This is a **really** good start but we'll need to be more succinct in our definition and a little less effusive with our praise of the tech. Ex: 

What is a managed service?  it is a service provided by a third party that can be consumed by application developers or operations teams. Managed services range from data storage, to monitoring, to analytics, or even text messages.

File: definitions/managed sevices.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Listed below are the main challenges Managed Services addresses and resolves.
```
Comment (JasonMorgan) [inline definitions/managed sevices.md:10]: Great writing but not consistent with the current glossary style.

File: definitions/managed sevices.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+### Problem it addresses
+
```
Comment (JasonMorgan) [inline definitions/managed sevices.md:13]: This whole section needs to be compressed. No section should be more than 3-4 sentences. Look at the commonalities between these benefits and try again. I want to re-emphasize that your writing style and enthusiasm is wonderful and I would love to get your definition into the next version of the glossary.

File: definitions/managed sevices.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+Vendor management:
+Facilitating and maintaining  relationships between your organization and vendors/partners, negotiating contracts, creating standards for the vendors, and finding the best available vendors,is too tedious of a job .
+
+But Managed Sevices eases everything . Lets find out how! 
+
+
+
+
+### How it helps
+  1.Managed services can make sure your firewalls are configured properly, administer identity and access management solutions, and verify the security of your data in the cloud.
```
Comment (JasonMorgan) [inline definitions/managed sevices.md:32]: We don't want bullet points or numbering. This needs to be one or two paragraphs and ideally around 3-4 sentences.

**Review by JasonMorgan — CHANGES_REQUESTED** [review]

File: definitions/managed sevices.md (hunk 1 above)
Comment (ShivamTyagi12345) [reply definitions/managed sevices.md:8]: @JasonMorgan I would gladly do the changes you are asking for. The next version would be comprising

- compressed paragraph, consistent with current glossary styling
- Won't sound like a Sales-representative(Hopefully)

File: definitions/managed sevices.md (hunk 4 above)
Comment (ShivamTyagi12345) [reply definitions/managed sevices.md:32]: Read

File: definitions/managed sevices.md (hunk 3 above)
Comment (ShivamTyagi12345) [reply definitions/managed sevices.md:13]: read

File: definitions/managed sevices.md (hunk 2 above)
Comment (ShivamTyagi12345) [reply definitions/managed sevices.md:10]: read

**Review by CathPag — APPROVED** [review]
This wasn't where it needed to be so we rewrote the entire entry. Thanks for giving it a try anyways.

**Review by JasonMorgan — APPROVED** [review]

**Comment by ShivamTyagi12345** [comment]
> This wasn't where it needed to be so we rewrote the entire entry. Thanks for giving it a try anyways.

Hy @CathPag .Thanks for reviewing,Was the definition not complete, or, it didn't meet a pre-set threshold ass set by other definitions?

However, I would like to improve in my next PR. Can I get started on [this issue](https://github.com/cncf/glossary/issues/194) now. I would really appreciate it if you mention the specific location where the entry needs to be made this time.

**Comment by CathPag** [comment]
@ShivamTyagi12345, sure. Have a look at the [new PR](https://github.com/cncf/glossary/pull/222).

**Pull request closed without merging**